### PR TITLE
fix(warehouse): deepsource fix for use of empty error string in errors.New

### PR DESCRIPTION
**Pull request by achettyiitr**
# Description

Added a fix for [deep-sources](https://deepsource.io/gh/rudderlabs/rudder-server/run/2284a648-4dd4-41a9-8dc3-c3d87c9e1fff/go/GO-W1024?listindex=0) issue `fix for use of empty error string in errors.New`.

## Notion Ticket

https://www.notion.so/rudderstacks/Test-cleanup-for-Datalakes-Integration-e6f8e3fd7b024b22aff23427d3e7d2c6

## Security

- [x] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

**Review by Jayachand — APPROVED** [review]

File: warehouse/upload_test.go [hunk 1]
```diff
@@ -160,7 +160,7 @@ var _ = Describe("Upload", Ordered, func() {
 	},
 		Entry(nil, []error{}, true),
 		Entry(nil, []error{&TableSkipError{}}, true),
-		Entry(nil, []error{errors.New("")}, false),
+		Entry(nil, []error{errors.New("some-error")}, false),
```
Comment (lvrach) [inline warehouse/upload_test.go:163]: How this test can pass regardless of the value of the error?

**Review by lvrach — APPROVED** [review]

File: warehouse/upload_test.go (hunk 1 above)
Comment (achettyiitr) [reply warehouse/upload_test.go:163]: `areAllTableSkipErrors` check if the error is of a specific type `TableSkipError`

Comment (achettyiitr) [reply warehouse/upload_test.go:163]: value doesn't matter.

Comment (achettyiitr) [reply warehouse/upload_test.go:163]: https://deepsource.io/gh/rudderlabs/rudder-server/run/2284a648-4dd4-41a9-8dc3-c3d87c9e1fff/go/GO-W1024?listindex=0

**Pull request merged**